docs/src/main/sphinx/release/release-449.md

```diff
@@ -24,23 +23,24 @@
 * Add support for [the `$partitions` system table](delta-lake-partitions-table). ({issue}`18590`)
 * Add support for reading from and writing to tables with
   [VACUUM Protocol Check](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#vacuum-protocol-check). ({issue}`21398`)
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+* Add support for configuring the request retry policy on the native S3
+  filesystem with the `s3.retry-mode` and `s3.max-error-retries` configuration
+  properties. ({issue}`21900`)
 * Automatically use `varchar` in struct types as a type during table creation
   when `char` is specified. ({issue}`21511`)
-* Improve performance of writing to Parquet files. ({issue}`22089`)
+* Improve performance of writing data files. ({issue}`22089`)
 * Fix query failure when the `hive.metastore.glue.catalogid` configuration
   property is set. ({issue}`22048`)
 
 ## Hive connector
 
 * Add support for specifying a catalog name in the Thrift metastore with the
-  `hive.metastore.thrift.catalog-name` configuration property. (`10287`)
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  `hive.metastore.thrift.catalog-name` configuration property. ({issue}`10287`)
+* Add support for configuring the request retry policy on the native S3
+  filesystem with the `s3.retry-mode` and `s3.max-error-retries` configuration
+  properties. ({issue}`21900`)
 * Improve performance of writing to Parquet files. ({issue}`22089`)
-* Fix failure when filesystem caching is enabled on Trino clusters with a single
-  node. ({issue}`21987`)
+* Allow usage of filesystem caching on Trino clusters with a single node. ({issue}`21987`)
```

Comment:
This better @raunaqmorarka

Reply:
It's better, would be more accurate to say `Allow usage of filesystem caching on Trino coordinator when "node-scheduler.include-coordinator" is enabled`

Reply:
Or are you saying it will cache on coordinator as well in all cases?

Reply:
Maybe then your suggestion is more correct.

Reply:
yes, this will enable caching on coordinator if coordinator is used as a worker in multi-node clusters as well

Reply:
ok .. will reword as suggested then